ember-toucan-core/src/components/form/checkbox-field.hbs

```diff
@@ -13,9 +13,11 @@
           aria-invalid={{if @error "true"}}
           aria-describedby="{{if @error field.errorId}}"
           id={{field.id}}
+          name={{@name}}
+          value={{@option}}
```

Comment:
More info for this at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#handling_multiple_checkboxes, but for a group of checkboxes, to differentiate one from the other, the `value` attribute is used directly.

In our case, `@value` is for the checked-state of the checkbox.  `@option` is used to define "this is a checkbox option" and on-click we can know which checkbox was checked or unchecked.  This is a little confusing as we now have a `value` attribute and a `@value` component argument, so I tried to add documentation below to assist.

Reply:
With this part: 
```hbs
value={{@option}}
```

1. Aren't we coupling the checkbox-field to `checkbox-group-field`. Do we want to do this? What if people want to have a single checkbox, and not a group.
2. `@option` seems optional, so perhaps this should be wrapped in an `if` statement if there is no option provided.

Reply:
NVM I re-read what you wrote and wrote below...